src/Style/TerminusStyle.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Pantheon\Terminus\Style;
+
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Terminus\Models\Workflow;
+
+class TerminusStyle extends SymfonyStyle
+{
+    const NORMAL_PROGRESS_FORMAT = " <bg=blue;fg=black> %message% </>\n <info>Progress: %bar% <info>%percent:3s%%</info> \n <info>Elapsed: %elapsed:6s% \n Estimated Time Remaining: %estimated:-6s%</info>";
+
+    /**
+     * TerminusStyle constructor.
+     *
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     */
+    public function __construct(InputInterface $input, OutputInterface $output)
+    {
+        ProgressBar::setFormatDefinition('normal', self::NORMAL_PROGRESS_FORMAT);
```

Comment:
We just need to be consistent about how Terminus styles are defined.  This seems fine to me; would we really want to define a whole system for configuring our style strings? Seems to me like the way you're doing it is the Symfony way (just set it in code).